account_invoice_currency/README.rst

```diff
@@ -25,16 +56,20 @@ Contributors
 * Jordi Esteve <jesteve@zikzakmedia.com>
 * Joaquín Gutierrez <joaquing.pedrosa@gmail.com>
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Antonio Espinosa <antonioea@antiun.com>
```

Comment:
Why your old Antiun account? You should remove it and leave only the current one, as you are no more active there. The trace of the company remains in authors, not here.